app/src/main/java/com/jerboa/ui/components/common/PopupItems.kt

```diff
@@ -0,0 +1,61 @@
+package com.jerboa.ui.components.common
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Gavel
+import androidx.compose.material.icons.outlined.Restore
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import com.jerboa.R
+import com.jerboa.communityNameShown
+import com.jerboa.datatypes.BanFromCommunityData
+import com.jerboa.personNameShown
+import it.vercruysse.lemmyapi.v0x19.datatypes.Person
+
+@Composable
+fun BanPersonPopupMenuItem(
```

Comment:
Externalized these to a common composable, since both post and comment items use them. Should maybe do the same thing for post / comment `Removes`